cli-toolchain/connectionWithKeyExamples.md

```diff
@@ -0,0 +1,493 @@
+# @connection With @key Use-cases
+
+Here are the 17 access patterns from https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-modeling-nosql.html that we can try and support in this example, and what the schema would look like:
+
+![alt text](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/images/AccessPatternList.png "Access Patterns")
+
+The following schema introduces the requisite keys and connections so that we can support all but the 17th access pattern depending on how one interprets the 17th pattern.
+
+
+```
+type Employee 
```

Comment:
The schema has some errors:
- Bool type is Boolean
- Integer type is Int
- Boolean values cannot be used as sort keys. error message when you try to deploy the schema.